Foundation/Array/Bitmap.hs

```diff
@@ -443,5 +446,23 @@ foldl' f initialAcc vec = loop 0 initialAcc
         | i .==# len = acc
         | otherwise  = loop (i+1) (f acc (unsafeIndex vec i))
 
+all :: (Bool -> Bool) -> Bitmap -> Bool
+all p bm = loop 0
```

Comment:
The rest of the code (elem etc.) all use `0`. Is it possible that this is already optimized?